src/data/content/index.yaml

```diff
@@ -58,7 +58,7 @@ components:
     big: true
     specular: true
     reversedMobile: false
-    title: Partecipa alla trasformazione digitale del Paese #C
+    title: Foobar #C
```

Comment:
⚠️❓ @bfabio

Reply:
See https://github.com/italia/designers.italia.it/pull/898#issuecomment-1735527293